setup.py

```diff
@@ -26,7 +26,15 @@
 def build_envd_if_not_found():
     if not os.path.exists("bin/envd"):
         logging.info("envd not found. Build from scratch")
-        errno = subprocess.call(["make", "build-release"])
+        try:
+            with open(".GIT_TAG_INFO") as f:
+                logging.info("Use build_tag from envd._version")
+                tag = f.read().strip()
+                errno = subprocess.call(
+                    ["GIT_TAG={}".format(tag), "make", "build-release"]
+                )
+        except:
+            errno = subprocess.call(["make", "build-release", ""])
```

Comment:
What is the purpose of the last empty string?

Reply:
fixed